README.md

```diff
@@ -32,13 +32,14 @@
 
 - **`sprocket analyzer`** runs Sprocket as a LSP server, which is useful for IDE integration.
 - **`sprocket check`** performs static analysis on a document or directory of documents.
+- **`sprocket completions`** generates shell completions for Sprocket.
 - **`sprocket config`** prints configuration values.
 - **`sprocket explain`** explains validation and lint rules supported by Sprocket.
 - **`sprocket format`** formats a document or directory of documents.
+- **`sprocket inputs`** writes template input file (JSON or YAML) for a task or workflow.
 - **`sprocket lint`** performs static analysis on a document or directory of documents with additional linting rules enabled (effectively a shortcut for `check --lint`).
 - **`sprocket run`** runs a task or workflow.
 - **`sprocket validate`** validates a set of inputs read from files or on the command line against a task or workflow.
-- **`sprocket completions`** generates shell completions for Sprocket.
```

Comment:
This was discussed somewhere I can't remember, but are we sure we want this sorted alphabetically? I'd slightly prefer this in order of "priority" (which is subjective). IMO:
- run
- analyzer
- check
- lint
- explain
- validate
- format
- config
- completions

We don't need to decide in this PR, but starting the conversation

Reply:
I think @claymcleod asked for an alphabetized list previously. It's worth a discussion, though. I do think a prioritized list might make sense here.